src/reader.js

```diff
@@ -1,5 +1,4 @@
-import fs from 'fs'
-import { promisify } from 'util'
+import { fsread, fs } from './promise-fs-opts.js'
```

Comment:
we don't need `fs` here anymore, can we just export a bool for this purpose? what what are the implications of `__browser` being set dynamically in here now based on this (haven't looked to remind myself), are we going to bump into other problems doing this?